spec/services/cache_service_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+
+describe CacheService do
+  let(:rails_cache) { Rails.cache }
+
+  describe "#cache" do
+    before do
+      rails_cache.stub(:fetch)
```

Comment:
We shouls tick to RSpec 3's syntax and use [allow](https://relishapp.com/rspec/rspec-mocks/v/3-9/docs/basics/allowing-messages). It does the same you want to do here with `have_received`.

Reply:
Updated in: https://github.com/openfoodfoundation/openfoodnetwork/pull/5317